package-generator/model/product_package.py

```diff
@@ -25,7 +25,7 @@
 import shutil
 import errno
 import os
-from util import utils
+from util import utils_file as utils
 from model.cookbook import Cookbook
 
 PACKAGES_FOLDER = "./../murano-apps"
```

Comment:
PACKAGE_FOLDERS is necessary 
 self.package_folder = (PACKAGES_FOLDER + "/" +
                               product.product_name + "/")
        self.package_classes = self.package_folder + "Classes/"
        self.package_classes_file = (self.package_classes +
                                     self.product.product_name + ".yaml")